essence-of-live-coding/test/Cell/Util.hs

```diff
@@ -108,5 +109,12 @@ test = testGroup "Utility unit tests"
         , Just (2 :: Int)
         , Nothing
         ]
-    }            
+    }
+    , testProperty "resampleListPar works as expected"
```

Comment:
Can you add a test (unit test is ok) to check whether growing and shrinking the input list works as expected?

By the way, you might like `CellSimulation` for such tests.

Reply:
added